client/public/locales/es/translation.json

```diff
@@ -309,7 +309,8 @@
   "toastr": {
     "success": {
       "added": "Éxito! {{what}} fue creado como un {{type}}.",
-      "applicationDeleted": "{{appIDCount}} aplicacione(s) descheda.",
+      "applicationDeleted": "{{appName}} aplicacione descheda.",
```

Comment:
IMHO we should drop messages that we cannot translate ourselves (unless we know Spanish :) ).
CC: @sjd78

Reply:
checked with a native Spanish speaker

Reply:
great!